Source/WebKit/GPUProcess/media/RemoteAudioDestinationManager.cpp

```diff
@@ -92,15 +93,15 @@ class RemoteAudioDestination final
     }
 
 #if PLATFORM(COCOA)
-    void audioSamplesStorageChanged(ConsumerSharedCARingBuffer::Handle&& handle, const WebCore::CAAudioStreamDescription& description, uint64_t numberOfFrames)
+    void audioSamplesStorageChanged(ConsumerSharedCARingBuffer::Handle&& handle, uint64_t numberOfFrames)
     {
         if (m_isPlaying) {
             stop();
             ASSERT(!m_isPlaying);
             if (m_isPlaying)
                 return;
         }
-        m_ringBuffer = ConsumerSharedCARingBuffer::map(description, numberOfFrames, WTFMove(handle));
+        m_ringBuffer = ConsumerSharedCARingBuffer::map(sizeof(Float32), m_numOutputChannels, numberOfFrames, WTFMove(handle));
```

Comment:
This requires this code to be kept in sync with AudioDestinationCocoa::getAudioStreamBasicDescription.
Maybe we should have a single routine for both code path that would create a description.